src/states/elder/mod.rs

```diff
@@ -1575,6 +1560,10 @@ impl Approved for Elder {
             });
 
             self.send_neighbour_infos();
+
+            if let Some(relocate_details) = self.chain.poll_relocation() {
```

Comment:
Again, should this happen even if not our section we update (Is NeighbourInfo causing trouble, probably not).

Reply:
See https://github.com/maidsafe/routing/pull/1866#discussion_r343024335